### DISCUSSION TITLE
Error : Your model is not yet fully trained

### DISCUSSION THREAD

**Discussion by RIdev-o9** [moj-analytical-services/splink#843] [General]
Hello, 
I am pretty new to splink and I am trying splink to check its fitment with respect to my problem statement (record matching). So, I have taken example dataset and code shown in splink introductory page (https://moj-analytical-services.github.io/splink/) . However I am getting error _Your model is not yet fully trained_. 
Also I tried on solving my problem statement (identify and cluster duplicate brands), it showed an error - _duckdb.ParserException: Parser Error: syntax error at or near ")"_ (screenshots below)

Can anyone please help me understand this why I am getting this error? Is there anything I am missing or doing in a wrong way ?

Python ver. - 3.8
splink ver. - 3.4.0
DuckDB ver. - 0.5.1

Sample Data and base code
![Code   Dataset](https://user-images.githubusercontent.com/115610216/195260280-ff721c9e-2b20-4ccc-9d27-f882c6d40981.png)

Error in running code on sample data 
![Error](https://user-images.githubusercontent.com/115610216/195260306-ce12e8bb-0d18-44bd-bc74-c9a71f379440.png)

Error in running code on my problem statement
![Error (In My Problem Statement)](https://user-images.githubusercontent.com/115610216/195262585-09ec0ff8-f804-4338-9caa-37e1135d6d1c.png)

**Comment by rapidAmbakar** [comment]
I got the same logs, it got resolve after doing this.
Before doing prediction on the model, try running linkerestimate_u_using_random_sampling, this should train the u values. i see you have implementated this in the code, still it's showing the these logs Interesting. Even i am trying to understand this Library

**Comment by RobinL** [comment]
The problem is that Splink assumes that you have multiple columns that need linking (e.g. first name, surname, date of birth...or for your data it could be brand name, address, market sector, revenue).  In general, Splink is not a good fit for a problem where you have a single column.  We should document this better somewhere!
